server2/Arcadia.Assistant/Arcadia.Assistant.UserFeeds/Models/Feed.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Arcadia.Assistant.UserFeeds.Models
+{
+    using Contracts.Interfaces;
+    using Contracts.Models;
+
+    public class Feed : IFeed
+    {
+        public FeedId Id { get; set; } = new FeedId(string.Empty);
+
+        public string Name { get; set; } = string.Empty;
```

Comment:
what name stands for? It's unclear. Also, is it needed?

Reply:
It seems used for feed name